test/test_torch.py

```diff
@@ -6066,13 +6066,6 @@ def test_neg(self, device):
         int_types = [torch.int, torch.short, torch.int8, torch.uint8]
         float_types = [torch.float, torch.double, torch.long]
 
-        # Tests bool tensor negation raises the correct error
-        self.assertRaisesRegex(
-            RuntimeError,
-            r"Negation, the `\-` operator, on a bool tensor is not supported. "
-            r"If you are trying to invert a mask, use the `\~` or `logical_not\(\)` operator instead.",
-            lambda: - torch.tensor([False, True], device=device))
-
```

Comment:
This assert was removed since `torch.neg` now supports Boolean tensors (implicitly promoted to default floating type) as inputs. This means: `torch.neg(torch.tensor(False))` now returns `-0.` .